### ISSUE TITLE
some issues about test AUC and ACC

### ISSUE THREAD

**Issue by Nullees** [Funzi/Diploma#1]
Hi, thanks for you example code about DKT.
Now i have some issues about it.
First, Your ”Pytorch.py“ is the implement of LSTM-DKT??
Second, i run pytorch.py and got the higher result(AUC and ACC) than your paper.Did you know it's reason? ?

**Comment by Nullees** [comment]
Epoch: 12,  Batch 124/125 Loss: 0.0015835739905014634
Correct 0   140273
Correct 1   645967
Predictions > 0.5   647029
==========================TRAIN SET==========================
RMSE for test set:0.04906
AUC for test set:0.99980
Pearson coef is: 0.9835774977965275
Accuracy is: 0.9967948717948718
####################################################
Correct 0   28225
Correct 1   131784
Predictions > 0.5   132025
==========================TEST SET==========================
RMSE for test set:0.04859
AUC for test set:0.99986
Pearson coef is: 0.9837518684708244
Accuracy is: 0.9968564268259911
